crates/primitives/src/lib.rs

```diff
@@ -176,64 +139,28 @@ pub trait BlockChunkExt: BlockTraceExt + BlockTraceRevmExt {
             hasher.update(tx_hash.as_slice())
         }
     }
-}
 
-/// Revm extension trait for Transaction
-pub trait Transaction {
-    /// get the raw tx type
-    fn raw_type(&self) -> u8;
-    /// check if the tx is l1 tx
-    fn is_l1_tx(&self) -> bool {
-        self.raw_type() == 0x7e
+    /// execution_results
+    fn execution_results(&self, _tx_id: usize) -> Option<&eth_types::l2_types::ExecutionResult> {
+        None
```

Comment:
remove this?  nowadays the sbv is quite stable, we have little need for the "post_check"?

Reply:
removed